design/sketch.webidl

```diff
@@ -106,15 +106,54 @@ interface WebGPUTexture {
 };
 
 // Sampler
-enum WebGPUFilterMode {
```

Comment:
Unintentional removal?

Reply:
It's on line 121 below.

Reply:
Oh, in that case, I assume you meant to remove the 3 lines below this?

Reply:
....yep.